gradle-plugin/src/test/resources/empty-test/build.gradle.kts

```diff
@@ -37,7 +37,7 @@ buildscript {
 plugins {
     java
     id("com.google.protobuf")
-    id("io.spine.proto-data") version "0.1.8"
+    id("@PROTODATA_PLUGIN_ID@") version "@PROTODATA_VERIONS@"
```

Comment:
There is a typo in `PROTODATA_VERIONS`.

It should probably be `PROTODATA_VERSION`. It's here and in all other places where it is used.